V3-Upgrade-Guide.md

```diff
@@ -81,23 +81,20 @@ The following sub-sections describe affected modules, functions and interfaces w
 
   ```ts
   .execute({
-    destinationName:
-    'DESTINATION',
+    destinationName: 'DESTINATION',
     useCache: true,
     isolationStrategy: 'tenant'
   })
   ```
 
-- The generic type parameter `JwtKeysT` in `JwtKeyMapping` is now narrowed to extend `string`.
```

Comment:
why delete this line?

Reply:
Because it is already there a few lines above